src/observer.ts

```diff
@@ -233,6 +242,7 @@ export class Observer {
     const referenceResolution = await getArnsResolution({
       host: this.referenceGatewayHost,
       arnsName,
+      nodeReleaseVersion: AR_IO_NODE_RELEASE,
```

Comment:
Let's pass this in when the observer is instantiated instead of referencing it directly.

Reply:
[`120dd04` (#29)](https://github.com/ar-io/ar-io-observer/pull/29/commits/120dd049c04a4369aeb0e53fcfd198858a70d04f)